src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Node/JsonNodeConverter.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Text.Json.Node;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal class JsonNodeConverter : JsonConverter<object>
+    {
+        public static JsonNodeConverter Default { get; } = new JsonNodeConverter();
+        public JsonArrayConverter ArrayConverter { get; } = new JsonArrayConverter();
+        public JsonObjectConverter ObjectConverter { get; } = new JsonObjectConverter();
+        public JsonValueConverter ValueConverter { get; } = new JsonValueConverter();
+        public ObjectConverter ElementConverter { get; } = new ObjectConverter();
```

Comment:
Is it worth it to instantiate these lazily and save on allocs?

Reply:
They are already lazy. The `JsonNodeConverterFactory` will only access these properties when given a JsonNode Type.

Reply:
I believe these properties will be initialized by the constructor.

Reply:
The properties will be initialized on first access (not the constructor)

Reply:
These particular properties will be initialized eagerly by the constructor:

https://sharplab.io/#v2:C4LglgNgNAJiDUAfAAgJgIwFgBQyDMABGgQMIEDeOB1RhA9gEYBWApgMbAEBiddFBAcxbAA3AQC+BALwEAdiwDuBRqw4AKAJQiqNfMubtOAIQCGAJ35DRE6XMX7VwTduw0CO6nuQAWAgFlNCg8JHHEgA